### PR TITLE
loading Fierfox

### PR DESCRIPTION
**Pull request by Dimitar91**
loading browser and search  something

### CODE REVIEW

**Comment by VladimirMilov** [comment]
Read about .gitignore file. Node modules should not be committed.

File: index.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+const { Builder, By, Key, util } = require("selenium-webdriver");
+async function examle() {
+    let driver = await new Builder().forBrowser("firefox").build();
```
Comment (VladimirMilov) [inline index.js:3]: extract the browser name into a constant. Its a good practise to have constants for generic strings that might be reused in the future.
Add this on the second line:
`const BROWSER = 'firefox'`

**Comment by VladimirMilov** [comment]
Always check the spelling of the words! 
Update PR's title message!
Every word that you have to write and its going to be public, please check it in google first!

**Comment by VladimirMilov** [comment]
you .gitignore file is empty, in case you didn't know.
Also remember what is the end goal: node modules shouldn't be visible here in the PR.

File: index.js (hunk 1 above)
Comment (Dimitar91) [reply index.js:4]: tanks for support.  I check cod and make the change.

Comment (VladimirMilov) [reply index.js:4]: Thanks for that you finally answered some of my comments!
And by the way the comment I made about you checking how every word is spelled also applies when you are writing comments. Can you please edit the message above to make it right?

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
+
+
```
Comment (VladimirMilov) [inline .gitignore:3]: Remove these empty lines

File: index.js (hunk 1 above)
Comment (VladimirMilov) [reply index.js:4]: now BROWSER is the name of a variable right? and `firefox` is the value, right?
Having these assumptions can we think what also needs to be changed on the 4th line?

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,4 @@
...
+/node_modules
```
Comment (VladimirMilov) [inline .gitignore:4]: Great!

File: index.js (hunk 1 above)
Comment (Dimitar91) [reply index.js:4]: > tanks for support. I check the file and make the change.

Comment (Dimitar91) [reply index.js]: I think this is the change.

File: index.js [hunk 4]
```diff
@@ -1,7 +1,7 @@
 const { Builder, By, Key, util } = require("selenium-webdriver");
 const BROWSER = "firefox"
 async function examle() {
-    let driver = await new Builder().forBrowser("firefox").build();
+    let driver = await new Builder().BROWSER().build();
```
Comment (VladimirMilov) [inline index.js:4]: the usage of the variable is not correct.

**Comment by Dimitar91** [comment]
I fixed this problem.

**Pull request merged**